docs/cookbook/New-NSX-V-WLD.md

```diff
@@ -0,0 +1,272 @@
+
+```json
+
+{
+"name": "sfo01w01-cl01",
+"networks": [
+{
+"type": "VSAN",
+"vlanId": 2240,
+"mtu": 9000,
+"subnet": "172.16.240.0",
+"mask": "255.255.255.0",
+"gateway": "172.16.240.253",
+"ipPools": [
+{
+"start": "172.16.240.5",
+"end": "172.16.240.100"
+}
+]
+},
+{
+"type": "VMOTION",
+"vlanId": 2236,
+"mtu": 9000,
+"subnet": "172.16.236.0",
+"mask": "255.255.255.0",
+"gateway": "172.16.236.253",
+"ipPools": [
+{
+"start": "172.16.236.5",
+"end": "172.16.236.100"
+}
+]
+}
+]
+}
+
+```
+
+So first off lets get a list of current Network Pools. To do this run the following cmdlet:
+
+```powershell
+Get-VCFNetworkPool
+```
+
+As expected this returns a single network pool.
+
+#### TIP
+You can manipulate the return data in several ways.
+
+To create a new network pool using the json we created earlier run the following:
+
+New-VCFNetworkPool -json .\SampleJSON\NetworkPool\addNetworkPoolSpec.json
+
+Now running Get-VCFNetworkPool should display 2 Network Pools
+
+### Commission Hosts
+
+Now that you have a network pool you can commission hosts and associate them with the network pool. For this you need the following json
+
+#### TIP 
+For this json you need the network pool name & ID. These were returned when the pool was created and also by Get-VCFNetworkPool
+
+```json
+
+[
+{
+"fqdn": "sfo01w01esx01.sfo01.rainpole.local",
+"username": "root",
+"storageType": "VSAN",
+"password": "VMw@re1!",
+"networkPoolName": "sfo01w01-cl01",
+"networkPoolId": "afd314f6-f31d-4ad4-8943-0ecb35c044b9"
+},
+{
+"fqdn": "sfo01w01esx02.sfo01.rainpole.local",
+"username": "root",
+"storageType": "VSAN",
+"password": "VMw@re1!",
+"networkPoolName": "sfo01w01-cl01",
+"networkPoolId": "afd314f6-f31d-4ad4-8943-0ecb35c044b9"
+
+},
+{
+"fqdn": "sfo01w01esx03.sfo01.rainpole.local",
+"username": "root",
+"storageType": "VSAN",
+"password": "VMw@re1!",
+"networkPoolName": "sfo01w01-cl01",
+"networkPoolId": "afd314f6-f31d-4ad4-8943-0ecb35c044b9"
+},
+{
+"fqdn": "sfo01w01esx04.sfo01.rainpole.local",
+"username": "root",
+"storageType": "VSAN",
+"password": "VMw@re1!",
+"networkPoolName": "sfo01w01-cl01",
+"networkPoolId": "afd314f6-f31d-4ad4-8943-0ecb35c044b9"
+}
+]
+
+```
+
+So to commission the 4 new hosts into my VCF inventory you run:
+
+Commission-VCFHost -json .\SampleJSON\Host\commissionHosts.json
+
+#### TIP 
+This returns a task id, which you can monitor by running the following until status=Successful:
+```Powershell
+Get-VCFTask -id b93e2bc7-627b-4f7c-980b-c12b3497c4ea
+```
+### Create a Workload Domain
+
+Once the commission hosts task is complete you can then create a workload domain using those hosts. Creating a workload domain also requires a json file. For this you need the id's of the hosts that you want to use. In VCF hosts that are available to be used in a workload domain have a status of UNASSIGNED_USEABLE so to find the id's of the hosts you want to add run the following
+
+#### TIP 
```

Comment:
Should there be some link breaks in here ?

Reply:
Displays ok on readthedocs